common/decisions/wc_loan_decisions.txt

```diff
@@ -0,0 +1,22 @@
+﻿get_loan_decision = {
```

Comment:
Make the format, names, effects, descriptions, ai checks, and values all similar to the holy order decision, including the repay part. There only need to be minor tweaks from that decision since all we are essentially doing here is giving an option to those without a holy order. A difference I would also make is that Goblins want to make more money out of it (so either they ask for repayment faster or ask for higher interest, idk if the original decision does that but I'm just giving an example)